book/website/reproducible-research/case-studies/statistical-methods-manuscript.md

```diff
@@ -0,0 +1,146 @@
+(rr:cs:statistical-methods-manuscript)=
+## A Statistical Methods Manuscript
+
+
+### Version control
+
+The git repository 
+[https://github.com/kkmann/sample-size-calculation-under-uncertainty](https://github.com/kkmann/sample-size-calculation-under-uncertainty)
+contains all code required to produce the manuscript 
+[arXiv:2006.15715](https://arxiv.org/abs/2006.15715)
+from scratch.
+For an in-depth explanation of the importanc eof version control for 
+reproducible research, see {ref}`rr-vcs`.
+
+
+### Research data management
+
+In this particular case, 
+{ref}`rr-rdm` data management aspects are not an issue since the 
+the manuscript is exclusively based on hypothetical examples and no
+external, protected data is required. 
+
+
+### Literate programming
+
+The manuscript itself is built using [LaTeX](https://www.latex-project.org/)
+from sources contained in the subfolder `latex/`. 
+Plain TeX files were prefered over literate programming solutions like 
+[knitr](https://github.com/rstudio/rmarkdown) for [R](https://www.r-project.org/)
+to facilitate the use of dedicated LaTeX editors like [Overleaf](https://www.overleaf.com/project).
+This means, however, that all figures used in the manuscript need to be 
+created separately. 
+A dedicated [Jupyter notebook](https://jupyter.org/)
+`notebooks/figures-for-manuscript.ipynb` combining code and rudimentary 
+descriptions is provided to that end. 
+
+
+### Reproducible software environment
+
+Although this means that all code required to compile the manuscript from scratch
+is available,
+it is not yet sufficient for true reproducibility.
+The just installing LaTeX, Jupyter, and R with the exact same specification 
+needed to run all code can be challenging. 
+To avoid this from keeping interested readers from experimenting with the code,
+a combination of the python package 
+[repo2docker](https://github.com/jupyter/repo2docker) and the free 
+[mybinderhub.org]() hosting service is used. 
+For details on these techniques, see the chapters on {ref}`rr-binder` and {ref}`rr-binderhub`.
+This allows interested individuals to start an interacitve version of the 
+repository with all required software preinstalled - in exactly the right
+versions!
+Note that it is possible to provide *version stable* binder links
```

Comment:
```suggestion
### Reproducible software environment

Although this means that all code required to compile the manuscript from scratch
is available in a self-contained software environment,
it is not sufficient for ensuring reproducibility.
Installing LaTeX, Jupyter, and R with the exact same specification 
needed to run all code can still be challenging for new learners. 
To avoid this from keeping interested readers from experimenting with the code,
a combination of the python package 
[repo2docker](https://github.com/jupyter/repo2docker) and the free 
[binderhub](https://mybinder.org/) hosting service is used. 
For details on these techniques, see the chapters on {ref}`Binder <rr-binder>` and {ref}`Binderhub <rr-binderhub>`.
This allows interested individuals to start an interactive version of the 
repository with all required software preinstalled - in exactly the right
versions!
Note that it is possible to provide *version stable* binder links
```